draft-gazdag-x509-hash-sigs.md

```diff
@@ -113,21 +141,16 @@ understood and depends only on the security of the underlying hash function. As
 such they can serve as an important building block for quantum computer
 resistant information and communication technology.
 
-The private key of HSS, XMSS and XMSS^MT is a finite collection of OTS keys,
-hence only a limited number of messages can be signed and the private key's
-state must be updated and persisted after signing to prevent reuse of OTS keys.
-Due to thise statefulness of the private key and the limited number of
-signatures that can be created, these signature algorithms might not be
-appropriate for use in interactive protocols. While the right selection of
-algorithm parameters would allow a private key to sign a virtually unbounded
-number of messages (e.g. 2^60), this is at the cost of a larger signature size
-and longer signing time. Since these algorithms are already known to be secure
-against quantum attacks, and because roots of trust are generally long-lived and
-can take longer to be deployed than end-entity certificates, these signature
-algorithms are more appropriate to be used in root and subordinate CA
-certificates. They are also appropriate in non-interactive contexts such as code
-signing. In particular, there are multi-party IoT ecosystems where publicly
-trusted code signing certificates are useful.
+The private key of HSS, XMSS and XMSS^MT is a finite collection of OTS keys, hence only a limited
+number of messages can be signed and the private key's state must be updated and persisted
+after signing to prevent reuse of OTS keys. Due to this statefulness of the private key
+and the limited number of signatures that can be created, these signature algorithms might
+not be appropriate for use in interactive protocols. While the right selection of algorithm
+parameters would allow a private key to sign a virtually unbounded number of messages (e.g. 2^60),
+this is at the cost of a larger signature size and longer signing time. Since these algorithms
+are already known to be secure against quantum attacks, and because roots of trust are generally
+long-lived and can take longer to be deployed than end-entity certificates, these signature
+algorithms are more appropriate to be used in some use case scenarios.
```

Comment:
Ich finde den Aufbau dieses Absatzes nicht stimmig:
Der 1. und 3. Satz beschreibt HBS und listet einige
Eigenschaften auf.
Der 2. und 4. Satz sagt etwas über mögliche use-cases.
Außerdem: Dass HBS quantensicher ist, steht schon im
vorherigen Abschnitt. Ich würde es hier nicht mehr
wiederholen, da es im "use cases-Kapitel auch erwähnt wird.
https://github.com/x509-hbs/draft-gazdag-x509-hash-sigs/pull/10/commits/95d5bd7914247daeac0bf3c6e03acd541ae8223f?di…
2/615.06.23, 16:26
First attempt to improve the description on the use cases. See issue 2 by exception42 · Pull Request #10 · x5…
Mein Gegenvorschlag:
The private key of HSS, XMSS and XMSS^MT is a finite collection
of OTS keys, hence only a limited
number of messages can be signed and the private key's state
must be updated and persisted
after signing to prevent reuse of OTS keys. While the right selection
of algorithm
parameters would allow a private key to sign a virtually unbounded
number of messages (e.g. 2^60),
this is at the cost of a larger signature size and longer signing time.
Due to the statefulness of the private key of HSS, XMSS and
XMSS^MT and the limited number of signatures that can be
created, these signature algorithms might not be appropriate for
use in interactive protocols. However, in some use case scenarios
the deployment of these signature algorithms may be appropriate.
Such use cases are described and discussed in the next section.